src/components/Trips.js

```diff
@@ -22,9 +32,17 @@ function Trips() {
               alt="destination"
```

Comment:
I realized you are relying on the size you pass to unsplash for the images size. 
You should set a size for your image using css